docs/CONTRIBUTING_GUIDE.md

```diff
@@ -0,0 +1,28 @@
+# Contributing to LibreLingo
+
+### Contribute to course material
+If you are interested in contributing to course development, please fill the [this](https://danielkantor196881.typeform.com/to/V00Paz) form. The project is in a very early stage right now, so you might not be able to contribute right away. Your work will be released in this GitHub repository and you will show up as a contributor here.
+
+
+### Development and other forms of contribution
+
+Check the latest **Development documentation** on [Read the Docs](https://librelingo.readthedocs.io/en/latest/)
+
+This project has 2 main components
```

Comment:
this is rather out of place here now, this is technical documentation

Reply:
I didn't understand this @kantord can you please elaborate?